### PR TITLE
Improve documentation about encoding option

### PR DESCRIPTION
**Pull request by Jhonatannunessilva**
This PR resolves https://github.com/elixir-explorer/explorer/issues/791

After this PR:

DataFrame.from_csv/2:

image

DataFrame.load_csv/2:

image

### CODE REVIEW

File: lib/explorer/data_frame.ex [hunk 1]
```diff
@@ -627,6 +630,7 @@ defmodule Explorer.DataFrame do
     * `:eol_delimiter` - A single character used to represent new lines. (default: `"\n"`)
     * `:backend` - The Explorer backend to use. Defaults to the value returned by `Explorer.Backend.get/0`.
     * `:lazy` - force the results into the lazy version of the current backend.
+    * `:encoding` - Encoding to use when reading the file. For now, the only possible values are `utf8` and `utf8-lossy`. The utf8-lossy option means that invalid utf8 values are replaced with � characters. (default: `"utf8"`)
```
Comment (josevalim) [inline lib/explorer/data_frame.ex:633]: @philss Should we make them atoms instead of regular strings?

Comment (josevalim) [inline lib/explorer/data_frame.ex:633]:
```suggestion
    * `:encoding` - Encoding to use when reading the file. For now, the only possible values are
      `utf8` and `utf8-lossy`. The utf8-lossy option means that invalid utf8 values are
      replaced with � characters. (default: `"utf8"`)
```

Comment (philss) [reply lib/explorer/data_frame.ex:633]: @josevalim yeah, I think so! 

@Jhonatannunessilva there is an example of what we want to do in https://github.com/elixir-explorer/explorer/pull/770
But if you prefer, we can do in another PR as well :)

Comment (Jhonatannunessilva) [reply lib/explorer/data_frame.ex:633]: No problem, I can do it in this PR

**Pull request merged**